wpt_usage_stats.py

```diff
@@ -18,13 +18,22 @@
     '.*/inspector-protocol/',
     # moved in crbug.com/667560
     '.*/inspector/',
+    # Expectations
     'third_party/WebKit/LayoutTests/FlagExpectations/',
     # TestExpectations, etc.
     'third_party/WebKit/LayoutTests/[^/]*$',
     # WPT_BASE_MANIFEST.json, etc.
     'third_party/WebKit/LayoutTests/external/[^/]*$',
     # lint.whitelist, etc.
     'third_party/WebKit/LayoutTests/external/wpt/[^/]*$',
+    # Baselines
+    'third_party/WebKit/LayoutTests/flag-specific/',
+    'third_party/WebKit/LayoutTests/platform/',
+    'third_party/WebKit/LayoutTests/virtual/',
+    '.*-expected\.(txt|png|wav)$',
```

Comment:
It was actually deliberate on my part to include changes to baselines. That's because for testharness.js tests, updating source code + a baseline at the same time is really using the test, and it counts as a use for cases where the test recently came in via import and is then fixed.

However, baselines also often have console message that get changed, or error messages changing. For js-test.js it's pretty common to use debug() or get stuff into the baseline which isn't strictly speaking part of the pass condition.

Overall, do you think ignoring baselines is better?